task2.java

```diff
@@ -0,0 +1,26 @@
+package com.Selenium.homework1;
+
+
+import java.util.*;
+
+
+import org.w3c.dom.ls.LSOutput;
+
+import java.util.*;
+
+public class task2 {
+    public static void main(String[] args) {
+        Integer[] masyv = {4, 2, 43, 54, 3, 2, 0, 6, 43, 2};
+
+        List<Integer> words = new ArrayList<Integer>();

```

Comment:
list of integers named words?
also do we actually need that list?

Reply:
fixed in 'task2.changes'